governance/xc-admin/packages/xc-admin-frontend/components/EditButton.tsx

```diff
@@ -0,0 +1,18 @@
+const EditButton = ({
+  editable,
+  onClick,
+}: {
+  editable?: boolean | null
```

Comment:
mmmmm what is the difference between `null` and `false`

Reply:
good catch will remove